### PR TITLE
feat: Ae coin transactions

### LINKED ISSUE #818: AE coin transactions
**Acceptance criteria**:
- A new tab called "Transactions" is visible in the AE coin details page
- The table displays "SpendTx" transactions and has the following columns:
  - Hash
  - Created Age (or Created Time)
  - From
  - To
  - Amount
- Each column of the table has tooltips

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #818 

## Demo

image

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)

### CODE REVIEW

**Comment by janmichek** [comment]
@michele-franchi one more implementation note.
To make things simple I just reused the transactions-panel which means that the 'type' column is shown.
I can still make it as proposed in initial requirements (without type column) but I would need to create dedicated components. Let me know what approach you like

File: src/utils/hints/aeCoinHints.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export const aeCoinHints = {
   aeCoin: 'æternity native coin.',
-  tokenSymbol: 'Reserved coin name.',
-  price: 'Price of the coin in AE and USD.',
+  tokenSymbol: 'Reserved token name.',
+  price: 'Price of the token in AE and USD.',
```
Comment (michele-franchi) [inline src/utils/hints/aeCoinHints.js:4]: I would keep coin, instead of token.

File: src/utils/hints/aeCoinHints.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
   totalSupply: 'Amount of tokens that will be mined.',
```
Comment (michele-franchi) [inline src/utils/hints/aeCoinHints.js:5]:
```suggestion
  totalSupply: 'Amount of coins in circulation.',
```

File: src/components/AeCoinTransactionsPanel.vue [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      <transactions-table-condensed
+        :transactions="transactions"
+        class="u-hidden-desktop"/>
+    </paginated-content>
+  </app-panel>
+</template>
+
+<script setup>
+import { useTransactionsStore } from '~/stores/transactions'
+import { isDesktop } from '~/utils/screen'
```
Comment (michele-franchi) [inline src/components/AeCoinTransactionsPanel.vue:19]:
```suggestion
import { useTransactionsStore } from '@/stores/transactions'
import { isDesktop } from '@/utils/screen'
```

**Review by michele-franchi — CHANGES_REQUESTED** [review]

File: src/utils/hints/aeCoinHints.js (hunk 1 above)
Comment (janmichek) [reply src/utils/hints/aeCoinHints.js]: all changed to 'coin'

**Review by michele-franchi — APPROVED** [review]

**Pull request merged**